package/MDAnalysis/coordinates/dcdtimeseries.pyx

```diff
@@ -109,18 +111,25 @@ def __read_timecorrel(object self, object atoms, object atomcounts, object forma
     cdef int index, numskip
     cdef int i, j
     cdef float unitcell[6]
+    cdef int remaining_frames = stop-start
+    numskip = 0
+    print n_frames
```

Comment:
remove `print` (note: I am commenting on individual commits, if this has been removed later, please ignore)